rest/handler.go

```diff
@@ -128,6 +128,7 @@ func makeHandler(server *ServerContext, privs handlerPrivs, accessPermissions []
 		err := h.invoke(method, accessPermissions, responsePermissions)
 		h.writeError(err)
 		h.logDuration(true)
+		h.logRESTCount()
```

Comment:
I wonder if the right place to put this is https://github.com/couchbase/sync_gateway/blob/master/rest/handler.go#L266

Because it will count requests before they happen - since continuous feed requests are long lived. I'm not sure how many people use them though, or if there are other long running requests. I'd consult with other team members on this.

Reply:
Agree - would be preferable to increment stat on incoming request, rather than completed request.